projects/plugins/jetpack/extensions/blocks/one-payment/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+import { InnerBlocks } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import icon from './icon';
+import variations from './variations';
+import { getIconColor } from '../../shared/block-icons';
+
+/**
+ * Style dependencies
+ */
+import './editor.scss';
+
+export const name = 'one-payment';
+export const title = __( 'Payments', 'jetpack' );
+export const settings = {
+	title,
+	description: __( 'Sell products and services or receive donations on your website', 'jetpack' ),
+	icon: {
+		src: icon,
+		foreground: getIconColor(),
+	},
+	category: 'earn',
+	keywords: [
+		_x( 'sell', 'block search term', 'jetpack' ),
```

Comment:
I forgot I did this, but I just added some keywords a while ago and didn't push. They are mostly taken directly from the blocks this block can add plus some more like 'checkout', and 'pay'. 

This is ok, but perhaps ideally those extra keywords would go back to the introduced blocks and this would pull keywords from them.

Reply:
Oh, that's an interesting point!
Thinking of something like this:
```js
keywords: [
  otherBlock.title,
  ...otherBlock.keywords,
  'custom-keywords'
],

// I think the title would be useful too.
// Maybe `...uniq( [ title, ...keywords ] )` in case the block repeats the title as keyword?
```

Given we are already importing the "other blocks" settings for the variations, it's reasonable to keep importing more info here as well.

IMHO we can do it here in advance to updating the Donations keywords.